dependencies.yaml

```diff
@@ -159,54 +178,54 @@ dependencies:
         packages:
           - &pylibraft_conda pylibraft==24.6.*
           - &rmm_conda rmm==24.6.*
-          - scikit-build-core>=0.7.0
       - output_types: requirements
         packages:
           # pip recognizes the index as a global option for the requirements.txt file
           # This index is needed for cudf and rmm.
           - --extra-index-url=https://pypi.nvidia.com
           - --extra-index-url=https://pypi.anaconda.org/rapidsai-wheels-nightly/simple
-      - output_types: [pyproject, requirements]
-        packages:
-          - scikit-build-core[pyproject]>=0.7.0
     specific:
       - output_types: [conda, requirements, pyproject]
         matrices:
           - matrix:
               cuda: "12.*"
             packages:
               - cuda-python>=12.0,<13.0a0
-          - matrix: # All CUDA 11 versions
+          - matrix:
+              cuda: "11.*"
             packages:
               - cuda-python>=11.7.1,<12.0a0
+          - matrix:
+            packages:
       - output_types: [requirements, pyproject]
         matrices:
           - matrix: {cuda: "12.*"}
             packages:
-              - pylibraft-cu12==24.6.*
-              - rmm-cu12==24.6.*
+              - pylibraft-cu12==24.6.*,>=0.0.0a0
+              - rmm-cu12==24.6.*,>=0.0.0a0
```

Comment:
Please verify that the `update-version.sh` script still works as intended. I think it might take the `.*` in the "find" pattern and trim off the rest of this line with the alpha spec.

https://github.com/rapidsai/cuml/blob/1609fcd03da81066d9d8a98ffdb741107e3f62f1/ci/release/update-version.sh#L65